libs/platform/src/lib/form/multi-input/multi-input.component.html

```diff
@@ -34,9 +34,9 @@
         [disabled]="disabled"
         [compact]="contentDensity === 'compact'"
         (keydown)="removeSelectedTokens($event)"
-        (addOnButtonClicked)="!mobile && addOnButtonClick()"
+        (addOnButtonClicked)="!mobile && addOnButtonClick($event)"
```

Comment:
Please remove `$event` here as we're not expecting parameters there.

Reply:
fixed